config/database.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+/*
+ * Here goes the illuminate/database component configuration. Once
+ * installed, the configuration below is used to configure your
+ * database component.
+ */
+return [
+    /*
+     * If true, migrations commands will be available.
+     */
+    'with-migrations' => true,
```

Comment:
do we need these?

Reply:
Nope, both `with-seeds` and `with-migrations` are now set to `false`.